Core/Solutions/Models/HaskellResponse.cs

```diff
@@ -0,0 +1,45 @@
+using System.Text.Json.Serialization;
+
+namespace Core.Solutions.Models;
+
```

Comment:
I assume mozart would return the the format of values no matter what language it provided in the future, why I would make a case for renaming the `HaskellResponse` class, to something on the lines of `SolutionRunnerResponse`

Reply:
Also sort of torn in the Mozart naming of the records in this file, as it suggest a tight cuppling to the solution runner service.
However, theoretically, were mozart to be replaced with another solution runner component, the responses are likely to be different, why a new response class would be required anyway, which demonstrates, why we in this aspect actually is coupled to mozart.